doc/source/methods/TreeSHAP.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>$F4$for --&gt; $F4$ for</p><p>Therefore, if a contribution is assigned to each leave --&gt; Therefore, if a contribution is assigned to each leaf</p><p>diagram below. --&gt; diagram above.</p><p>"one has to bear in mind that, for example, one needs" - too many filler words...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>